commit 301aebfa71143fb610eb7b816376a8c1f3e07208
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Sun Feb 10 18:01:33 2013 +0000

    configure: Add -lX11 required to link the legacy libI810XvMC.so
    
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/configure.ac b/configure.ac
index 3a4b6db..988cf51 100644
--- a/configure.ac
+++ b/configure.ac
@@ -416,7 +416,7 @@ AM_CONDITIONAL(DRI2, test "x$DRI2" = xyes)
 AC_MSG_RESULT([$DRI2])
 
 if test "$XVMC" = yes; then
-	PKG_CHECK_MODULES(XVMCLIB, [xvmc dri2proto], [], [XVMC=no])
+	PKG_CHECK_MODULES(XVMCLIB, [xvmc dri2proto x11], [], [XVMC=no])
 	PKG_CHECK_MODULES(XCB, [x11-xcb xcb-dri2 xcb-aux], [], [XVMC=no])
 fi
 AC_MSG_CHECKING([whether to include XvMC support])
